example/README.md

```diff
@@ -0,0 +1,25 @@
+## Quick Guide to running the terraform module
+
```

Comment:
steps to create the az credentials etc are not here? the steps went through yesterday

Reply:
added